### DISCUSSION TITLE
Retro Replay support (32k extra RAM)

### DISCUSSION THREAD

**Discussion by lubber-de** [frntc/Sidekick64#62] [Ideas]
Unfortunately it seems the widely spread retro replay cartridge is not supported. Only the original Action replay which only provides 8k extra RAM

Are there technical limits which prevent supporting retro replay as well?

**Comment by frntc** [comment]
So far I didn't have a closer look at the retro replay/nordic replay since it's still being produced and sold commercially.

If I remember correctly, it uses a RAM banking scheme which it not supported, but I have to check...
